cmd/prometheus/config.go

```diff
@@ -190,45 +187,6 @@ func init() {
 		"Local storage engine. Supported values are: 'persisted' (full local storage with on-disk persistence) and 'none' (no local storage).",
 	)
 
-	// Remote storage.
-	cfg.fs.StringVar(
```

Comment:
I'd rather see the support of an experimental feature broken after an update than the start of my server. In the end, responsible operators should just read the changelog.